app/templates/components/notification-dropdown.hbs

```diff
@@ -10,20 +10,22 @@
       {{/link-to}}
     </div>
     <div class="ui basic right aligned segment weight-400">
-      <a href="#">{{t 'Mark all as Read'}}</a>
+      <a href="#" {{action 'markAllRead'}}>{{t 'Mark all as Read'}}</a>
     </div>
   </div>
   <div class="ui fluid link celled selection list">
     {{#each notifications as |notification|}}
```

Comment:
In the list of notifications displayed, there is no way of a use knowing if a notification is unread or read. So ensure there is some indication. Take inspiration from sites like facebook.